docker-compose.yml

```diff
@@ -61,6 +62,7 @@ services:
     - "polkadot-volume:/polkadot"
 
   astar:
+    platform: linux/x86_64
```

Comment:
For public images that doesn't support aarch64 (like astar and polkadot), I have to explicit declare the platform architecture, Apple Silicon Docker is able to execute x86-64 containers with a performance penalty, but nothing too bad.